include/oneapi/dpl/pstl/glue_algorithm_ranges_defs.h

```diff
@@ -71,6 +89,26 @@ oneapi::dpl::__internal::__enable_if_execution_policy<_ExecutionPolicy,
                                                       oneapi::dpl::__internal::__difference_t<_Range1>>
 find_first_of(_ExecutionPolicy&& __exec, _Range1&& __rng1, _Range2&& __rng2);
 
+// [alg.adjacent_find]
+
+template <typename _ExecutionPolicy, typename _Range>
+oneapi::dpl::__internal::__enable_if_execution_policy<_ExecutionPolicy, oneapi::dpl::__internal::__difference_t<_Range>>
+adjacent_find(_ExecutionPolicy&& __exec, _Range&& __rng);
+
+template <typename _ExecutionPolicy, typename _Range, typename _BinaryPredicate>
+oneapi::dpl::__internal::__enable_if_execution_policy<_ExecutionPolicy, oneapi::dpl::__internal::__difference_t<_Range>>
+adjacent_find(_ExecutionPolicy&& __exec, _Range&& __rng, _BinaryPredicate __pred);
```

Comment:
This algorithm is not covered by tests. Do you plan to add it for this patch?

Reply:
This algorithm is not targeted on the closer release. So, there is no a test so far.

Reply:
I see. So I'd suggest to remove this algos from the patch or open a separate issue regarding adding tests since we can forget about the gap later.